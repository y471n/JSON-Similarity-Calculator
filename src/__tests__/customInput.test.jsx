import React from "react";
import { shallow } from "enzyme";
import CustomInput from "../components/customInput";

it("Should render a textarea with given title", () => {
  const wrapper = shallow(<CustomInput value="hello" />);
  expect(wrapper).toMatchSnapshot();
  const textarea = wrapper.find("textarea");
  expect(textarea).toBeDefined();
});
