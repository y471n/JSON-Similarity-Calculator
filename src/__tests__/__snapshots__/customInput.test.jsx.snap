// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a textarea with given title 1`] = `ShallowWrapper {}`;
